bindings/cs/Protean-cs/Protean.Net/Properties/AssemblyInfo.cs

```diff
@@ -10,7 +10,6 @@
 [assembly: AssemblyProduct("Protean.Net")]
 [assembly: AssemblyTrademark("")]
 [assembly: AssemblyVersion("2.0.0.0")]
```

Comment:
No:
https://codeblog.jonskeet.uk/2016/03/
https://codingforsmarties.wordpress.com/2016/01/21/how-to-version-assemblies-destined-for-nuget/

Keeping this as Major.0.0.0 avoids various dependency issues. We vary the AssemblyFileVersion and AssemblyInformationalVersion.